include/itkCuberilleImageToMeshFilter.hxx

```diff
@@ -190,13 +190,13 @@ CuberilleImageToMeshFilter<TInputImage, TOutputMesh, TInterpolator>::GenerateDat
         const auto   diff = vindex - index;
         size_t       offsetID = 7 - (diff[0] + diff[1] * 2 + diff[2] * 4);
         const auto   components = this->m_LabelsArray.at(bitmaskID);
-        const auto   numComponents = (*std::max_element(components.begin(), components.end())) + 1;
         const auto   component = components.at(offsetID);
         unsigned int look = (i < 4) ? look0 : look1; // First four are first slice
         if (!lookup[look].GetVertex(vindex[0], vindex[1], component, v[i]))
         {
           // Vertex was not in lookup, create and add to lookup
           v[i] = nextVertexId;
+          const auto numComponents = (*std::max_element(components.begin(), components.end())) + 1;
```

Comment:
Is this move into the for loop intentional?

Reply:
Yes--I just realized that there's no reason to calculate the number of components unless we're going to make a new vertex.  Unrelated, very minor optimization.